scripts/microbe_stage/microbe_operations.as

```diff
@@ -1138,36 +1158,41 @@ void kill(CellStageWorld@ world, ObjectID microbeEntity)
     microbeSceneNode.Marked = true;
 }
 
+// Default version of alterSpeciesPopulation that takes an ObjectID
 void alterSpeciesPopulation(CellStageWorld@ world, ObjectID microbeEntity, int popChange)
 {
-    MicrobeComponent@ microbeComponent = cast<MicrobeComponent>(
-        world.GetScriptComponentHolder("MicrobeComponent").Find(microbeEntity));
+    MicrobeComponent@ microbeComponent = getMicrobeComponent(world, microbeEntity);
     SpeciesComponent@ ourSpecies = getSpeciesComponent(world, microbeEntity);
+    alterSpeciesPopulation(world, ourSpecies, microbeComponent, popChange);
+}
 
+void alterSpeciesPopulation(CellStageWorld@ world,
+                            SpeciesComponent@ ourSpecies,
```

Comment:
This type of parameter aligning isn't the style we are using. I'm going to accept this pull request anyway, this can be fixed later. I'll create an issue at some point about all the minor comments I have made over the past month.